lighty-core/dependency-versions/pom.xml

```diff
@@ -30,6 +30,11 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
```

Comment:
do we need this to main?

Improve commit msg to contain version.

Reply:
Commit messages changed.